pallets/crowdloans/src/lib.rs

```diff
@@ -153,44 +153,38 @@ pub mod pallet {
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
         /// New vault was created
-        /// [para_id, ctoken_id]
-        VaultCreated(ParaId, AssetIdOf<T>),
+        /// [para_id, vault_id, ctoken_id, phase, contribution_strategy, cap, end_block, trie_index]
+        VaultCreated(
+            ParaId,
+            u32,
```

Comment:
I'm thinking that we can define two alias to make things more clear. 

```
pub type TrieIndex = u32;
pub type VaultId = u32;
```

we can put them inside primitives